File: LeetCode/Simulation/Python/Leetcode1598-crawler-log-folder.py
class Solution:
    def minOperations(self, logs: List[str]) -> int:
        stp, n = 0, len(logs)
        for i in range(n):
            if logs[i] == './' or (logs[i] == '../' and stp == 0): continue
            if logs[i] == '../': stp -= 1
            else: stp += 1
        return stp

File: LeetCode/Simulation/Golang/Leetcode1598-crawler-log-folder.go
package sbgo

func minOperations(logs []string) int {
	stp, n := 0, len(logs)
	for i := 0; i < n; i++ {
		if logs[i] == "./" || (logs[i] == "../" && stp == 0) {
			continue
		}
		if logs[i] == "../" {
			stp--
		} else {
			stp++
		}
	}
	return stp
}
